intellij/src/saros/intellij/editor/EditorManager.java

```diff
@@ -248,9 +253,13 @@ public void resourcesAdded(final IProject project) {
          * transmitted, a second editor activated activity is send for the locally active editor to
          * correctly set the active editor in the remote user editor state for the local user.
          */
-        private void sendAwarenessInformation() {
+        private void sendAwarenessInformation(@NotNull User joinedUser) {
```

Comment:
I would not use `joinedUser` . It suggest that the user has joined a session which is not the case. Either use `user` or `toUser`

Reply:
Done.

I confused this for a second as, currently in Saros/I, the project negotiation always coincides with the session negotiation. But this is only the case as long as you can't add resources to a running session.